debian/control

```diff
@@ -1,15 +1,15 @@
 Source: hw-probe
-Section: utils
 Priority: optional
+Section: utils
 Maintainer: Mikhail Novosyolov <mikhailnov@dumalogiya.ru>
 Build-Depends: debhelper (>=9), perl-base (>= 5), make
-Standards-Version: 4.0.0
+Standards-Version: 3.9.6
```

Comment:
Why not latest 4.1.3?

Reply:
![deepinscreenshot_select-area_20180114173032](https://user-images.githubusercontent.com/15802528/34916957-bf3be046-f950-11e7-8d91-53cd6ac48b7b.png)
This spec at 3.9.6 seems to be comptible with debian >=oldstable

Reply:
https://tracker.debian.org/pkg/debian-policy